frontend/app/(dashboardLayout)/equity/cap_table/page.tsx

```
@@ -157,9 +159,10 @@ export default function CapTable() {
   );
 
   const shareClassesTable = useTable({ data: shareClassesData, columns: shareClassesColumns });
-
+  const currentLink = navLinks(user, company).find((link) => link.route === pathname);
   return (
-    <EquityLayout>
+    <>
+      {currentLink && <PageHeader currentLink={currentLink} />}
```

ESLint

[error] Potential leaked value that might cause unintentionally rendered values or rendering crashes

(react/jsx-no-leaked-render)